python/lsst/analysis/tools/actions/vector/selectors.py

```diff
@@ -370,6 +370,7 @@ def __call__(self, data: KeyedData, **kwargs) -> Vector:
     def setDefaults(self):
         self.selectWhenFalse = [
             "{band}_pixelFlags_edge",
+            "{band}_pixelFlags_nodata",
```

Comment:
Do you mind adding `super().setDefaults()` above this line while you're at it?

Reply:
Added into the `setDefaults` for `SkyObjectSelector`, `SkySourceSelector`, and `GoodDiaSourceSelector`. How does that read now @arunkannawadi?

Reply:
It reads great now. Thanks, Lee